<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassShowResult</class>
 <widget class="QWidget" name="ClassShowResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCommandLinkButton" name="commandLinkButtonOne">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditOne">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>60</y>
     <width>371</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelOne">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#ffba4b;&quot;&gt;游戏成绩&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonTwo">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/circulation.png</normaloff>:/res/circulation.png</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonThree">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空统计</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
